config.json.example

```diff
@@ -5,10 +5,11 @@
     86477779717066752
   ],
   "prefix": [
-    "!"
+    "!",
+    "?"
```

Comment:
Wait, why is there a question mark here?

Reply:
More common prefixes:)

Reply:
Removed it